src/RSCoin/Core/Communication.hs

```diff
@@ -286,6 +289,21 @@ getStatisticsId =
         (L.logDebug . sformat ("Statistics id is " % int)) $
     callBank $ P.call (P.RSCBank P.GetStatisticsId)
 
+addMintetteUsingPermission
+    :: WorkMode m
+    => SecretKey
+    -> String
```

Comment:
Instead of passing `String` and `Int` you'd better pass `RSCoin.Core.Types.Mintette` which consists of exactly `String` and `Int`.